inlong-dashboard/src/pages/Clusters/NodeEditModal.tsx

```diff
@@ -83,11 +84,23 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
         type: 'input',
         label: 'IP',
         name: 'ip',
+        rules: [
+          {
+            pattern: rulesPattern.ip,
+            message: i18n.t('pages.Clusters.Node.IpRule'),
+          },
+        ],
       },
       {
         type: 'input',
```

Comment:
use `inputnumber`, set `min` and `max`

Reply:
https://github.com/apache/inlong/blob/master/inlong-dashboard/src/components/MetaData/DataSourcesBinLog.ts#L40